src/contrib/mlir/tools/ngraph-opt/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# ******************************************************************************
+# Copyright 2017-2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set(LIB_LIBS
+    MLIRPass
+)
+
+add_library(ngraph_opt
```

Comment:
Do you need the library ? Why not just the executable ? Also, if you must, place rename to something more clear `ngraph_opt_lib` , `ngraph_opt`

Reply:
I did the rename. I'm following the same approach as for mlir-opt https://github.com/tensorflow/mlir/blob/master/tools/mlir-opt/CMakeLists.txt
I assumed there was some benefit in linking in two steps but I don't know.